src/utils/VirtualModule.ts

```diff
@@ -30,7 +30,10 @@ function getNodeModulesDir() {
   return cachedNodeModulesDir;
 }
 
-export const virtualPackageName = '__mf__virtual';
+export function getVirtualPackageName(): string {
+  const options = getNormalizeModuleFederationOptions();
+  return options?.virtualModuleDir || '__mf__virtual';
```

Comment:
we already have the fallback [here](https://github.com/module-federation/vite/pull/305/files#diff-abc6cd67fc307307789c9cacee7765bf773360586663d0bc333eeb55766f8830R410) so this is not necessary I guess